.github/workflows/daily-csv-upload.yaml

```diff
@@ -0,0 +1,69 @@
+name: Upload current-full.csv and current-federal.csv
+run-name: Upload current-full.csv and current-federal.csv for branch ${{ github.head_ref }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Which environment do you wish to load reports for?
+        options:
+          - stable
+          - staging
+          - development
+          - ky
+          - es
+          - nl
+          - rh
+          - za
+          - gd
+          - rb
+          - ko
+          - ab
+          - bl
+          - rjm
+          - dk
+  # TODO - uncomment after #1403 is finished
+  #schedule:
+    # Runs every day at 5 AM UTC.
+    # - cron: "0 5 * * *"
+
+jobs:
+  variables: 
+    outputs:
+      environment: ${{ steps.var.outputs.environment }}
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Setting global variables
+        uses: actions/github-script@v6
+        id: var
+        with:
+          script: |
+            const environment = (github && github.event && github.event.inputs) ? github.event.inputs.environment : 'za';
```

Comment:
"za" will eventually be changed to "stable". It is set as za for now to avoid breaking things if a bug causes some issue